newIDE/app/src/EventsSheet/InlinePopover.js

```diff
@@ -83,7 +84,14 @@ export default function InlinePopover(props: Props) {
           // Note that the content of the popover can capture the event
           // and stop its propagation (for example, the GenericExpressionField
           // when showing autocompletion), which is fine.
+
+          // When the event match with Escape key changes are canceled.
           if (shouldCloseOrCancel(event)) {
+            props.onRequestClose(event);
+          }
+
+          // When event match with ctrlOrCmd + Enter changes are applied.
+          if (shouldSubmit(event)) {
             props.onRequestClose();
```

Comment:
The comment is not really aligned with what the code does. If I read the code, I can just see 
"If I should submit (i.e: if Ctrl + Enter was pressed), then I should close".